bedrock/security/templates/security/older-vulnerabilities.html

```diff
@@ -715,7 +715,7 @@ <h3 id="nov-2003">November 2003 Update</h3>
       <td valign="top">60</td>
       <td valign="top">Networking<br>
       </td>
-      <td style="vertical-align: top;">1.5 1.4.2</td>
+      <td valign="top">1.5 1.4.2</td>
```

Comment:
The alignment on these table cells is inconsistent throughout the page. I'd vote for removing the `valign`s everywhere rather than replacing these random inline styles. Maybe just set `td { vertical-align: top; }` for the whole table?